src/edfi-performance-test/edfi_performance_test/tasks/pipeclean/v5/assessment.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+
+from edfi_performance_test.tasks.pipeclean.assessment import (
+    LearningObjectivePipecleanTest,
+    LearningStandardPipecleanTest,
+)
+from edfi_performance_test.tasks.pipeclean.ed_fi_pipeclean_test_base import (
+    EdFiPipecleanTestBase,
+)
+
+
+class SkipLearningObjectivePipecleanTest(LearningObjectivePipecleanTest):
+    @classmethod
+    def skip_all_scenarios(cls):
+        return True
+
+
+class SkipLearningStandardPipecleanTest(LearningStandardPipecleanTest):
```

Comment:
Do we need to skip it and re add it again in LearningStandardPipecleanTestV5?

Reply:
Basically because in version 5 contentStandard is added as required data

Reply:
> Do we need to skip it and re add it again in LearningStandardPipecleanTestV5?

basically because in version 5 contentStandard is added as required data